Test_Cases_main.cpp

```diff
@@ -0,0 +1,56 @@
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "test/catch.hpp"
+#include <iostream>
+#include "Current_SamplingFunctionality.hpp"
+#include "Test_Cases_main.hpp"
+#include "ADC_To_AmpConverter.hpp"
+using namespace std;
+
+// As a first step, only considering to detect continuous ranges in the samples
+TEST_CASE("PASSING TEST CASE 1") {
```

Comment:
Use the TEST_CASE string to express functionality. E.g., instead of "PASSING TEST CASE 1", you can specify "detects two consecutive current samples"
When you see all the test cases at the end, you will know if you have tested everything, or if some possibilities are left out.